plugin-build/src/test/kotlin/io/sentry/android/gradle/instrumentation/database/sqlite/AndroidXSQLiteDatabaseVisitorTest.kt

```diff
@@ -0,0 +1,44 @@
+package io.sentry.android.gradle.instrumentation.database.sqlite
+
+import junit.framework.TestCase.assertEquals
+import org.junit.Test
+import org.objectweb.asm.ClassReader
+import org.objectweb.asm.ClassWriter
+import org.objectweb.asm.Opcodes
+import org.objectweb.asm.util.CheckClassAdapter
+import java.io.FileInputStream
+import java.io.PrintWriter
+import java.io.StringWriter
+
+class AndroidXSQLiteDatabaseVisitorTest {
+
+    @Test
+    fun `instrumented FrameworkSQLiteDatabase class passes Java verifier`() {
```

Comment:
if all the other test methods are kinda similar to this one, we could have a TestUtil class that does the job, eg, taking a `String className` and returns a pair of classVisitor and classWriter byte array

Reply:
not necessary for this PR unless we add more tests

Reply:
Yep I will convert it into a parameterized test I think, when I instrument other classes 👍